IMAGE_SCANNING.md

```diff
@@ -7,43 +7,48 @@ Scheduled actions only run on the `master` repository branch so will run once, r
 
 Both workflows read settings from the file [JOB.env](JOB.env) to ensure the same Java, Debian, and Defra versions are used during the image scan.
 
-Scans are performed by the [Anchore Engine CLI Tools](https://github.com/anchore/ci-tools) inline script using the policy file [anchore-policy.json](anchore-policy.json).
-Details on the policy configuration and exclusions can be found in [POLICY_CONFIGURATION.md](POLICY_CONFIGURATION.md).
+Scans are performed by [Anchore Grype](https://github.com/anchore/grype) using the configuration file [.grype.yaml](.grype.yaml) via the [Github Anchore Scan Action](https://github.com/anchore/scan-action).
+
+The scan is configured to fail on vulnerabilities of `medium` or higher.
+
+Details on the configuration file and exclusions can be found in [POLICY_CONFIGURATION.md](POLICY_CONFIGURATION.md).
 
 ## Addressing vulnerabilities
 
-If the Anchore scan finds a vulnerability the scan will fail and a report will be stored as an artifact against the failed GitHub [Action](https://github.com/DEFRA/defra-docker-java/actions).
+If the Grype scan finds a vulnerability the scan will fail and a report will be stored as an artifact against the failed GitHub [Action](https://github.com/DEFRA/defra-docker-java/actions).
```

Comment:
This isn't true if you are using the `upload-sarif` rather than `upload-artifact`. I'd be open to switching to it if there is a way to the code scanning report, but it seemed light on details when I tried.

Reply:
I've popped this back to upload-artifact.